apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.html

```diff
@@ -1,36 +1,118 @@
+                {{attributes.get(attribute)[1]}}
+              </td>
+            </ng-container>
+            <tr
+              *matRowDef="let attribute; columns: displayedColumns;"
+              mat-row
+            >
+            </tr>
+          </table>
+        </mat-card-content>
+      </mat-card>
+    </div>
+    <div class="align-cards">
+      <mat-card class="mat-elevation-z3">
+        <mat-card-header>
+          <mat-card-title>
+            <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.MEMBERSHIP' | translate}}</h1>
+          </mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
+          <table mat-table [dataSource]="membershipDataSource">
+            <ng-container matColumnDef="attName">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="font-weight-bold" mat-cell>
+                {{attribute}}:
+              </td>
+            </ng-container>
+            <ng-container matColumnDef="attValue">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let attribute" class="column-center" mat-cell>
+                <div *ngIf="attribute === 'Status'">
+                  <mat-icon class="vert-top mr-1" [color]="statusIconColor">
+                    {{statusIcon}}
+                  </mat-icon>
+                  <b class="vert-top">
+                    {{member.status | transformMemberStatus}}
+                  </b>
+                </div>
+                <div *ngIf="attribute === 'Expiration'">
+                  <i class="column-center">
+                    {{expiration}}
+                  </i>
+                  <button
+                    *ngIf="authResolver.isThisVoAdmin(vo.id)"
+                    (click)="changeExpiration()"
+                    mat-icon-button>
+                    <mat-icon>edit</mat-icon>
+                  </button>
+                </div>
+              </td>
+            </ng-container>
+            <tr
+              *matRowDef="let attribute; columns: displayedColumns;"
+              mat-row
+            >
+            </tr>
+          </table>
+        </mat-card-content>
+      </mat-card>
+    </div>
+    <div class="align-cards" *ngIf="member.sponsored">
+      <mat-card class="mat-elevation-z3">
+        <mat-card-header>
+          <mat-card-title>
+            <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.SPONSORED_BY' | translate}}</h1>
+          </mat-card-title>
+        </mat-card-header>
+        <mat-card-content *ngIf="sponsors.length !== 0">
+          <h5>{{'MEMBER_DETAIL.OVERVIEW.SPONSORS' | translate}}:</h5>
+          <table class="w-100" mat-table [dataSource]="sponsorsDataSource">
+            <ng-container matColumnDef="sponsors">
+              <th *matHeaderCellDef mat-header-cell></th>
+              <td *matCellDef="let sponsor" class="column-center" mat-cell>
+                {{sponsor}}
```

Comment:
```suggestion
                {{sponsor.user | userfullname}}
```
without this in the table it showed me "[ object ]"
![Uploading image.png…]()